src/App.tsx

```diff
@@ -128,6 +128,38 @@ function App() {
           >
             Buy an NFT
           </button>
+          <button
+            style={{
+              marginRight: "8px",
+            }}
+            onClick={async () => {
+              const operationList =
+                await beaconWallet.requestSimulatedProofOfEvent();
+
+              console.log("Operations:", atob(operationList));
+
+              const preapply = await Tezos.rpc.preapplyOperations(
+                JSON.parse(atob(operationList)),
+              );
+
+              console.log("Preapply response:", preapply);
+
+              if (
+                preapply[0].contents.every(
+                  (transaction) =>
+                    // @ts-expect-error - We know the type that'll be returned
+                    transaction.metadata.internal_operation_results[0].result
+                      .status === "applied",
+                )
+              ) {
+                alert("Simulated Proof Of Event succeeded");
```

Comment:
```suggestion
                console.log("The emitted proof of event:", preapply[0].contents[2].metadata.internal_operation_results[2]);
                alert("Simulated Proof Of Event succeeded");                
```
add a line to show event, that we really get.